config/Dockerfiles/tests.d/aws/07_aws_ec2_template

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash -xe
 
 # Verify the aws-ec2 provisioning with template data
-# distros.exclude: fedora28
+# distros.exclude: fedora29
```

Comment:
Do we need to run this on centos7 and fedora30?

Reply:
quota problems!